plugins/hls-retrie-plugin/src/Ide/Plugin/Retrie.hs

```diff
@@ -167,10 +165,11 @@ data RunRetrieParams = RunRetrieParams
     restrictToOriginatingFile :: Bool
   }
   deriving (Eq, Show, Generic, FromJSON, ToJSON)
+
 runRetrieCmd :: CommandFunction IdeState RunRetrieParams
 runRetrieCmd state token RunRetrieParams{originatingFile = uri, ..} = ExceptT $
   withIndefiniteProgress description token Cancellable $ \_updater -> do
-    runExceptT $ do
+    _ <- runExceptT $ do
```

Comment:
This is definitely suspicious: wouldn't we want to send the failure to client instead of ignoring it?

Reply:
Or we can atleast log the error?

Reply:
On second thought it's actually sending some errors notifications to the client within the block if errors occur, so this is probably ok.